### ISSUE TITLE
Unable to delete assignee of an issue

### ISSUE THREAD

**Issue by stolp** [taskadapter/redmine-java-api#325]
I have tried several approaches for deleting an assignee of an issue.

issue.setAssigneeId(null);
issue.setAssigneeName(null);
issue.setAssigneeName("");

None of these attempts resets the assignee in the remote Redmine issue.

What am I missing here? Is this a bug?

I am using com.taskadapter:redmine-java-api:4.0.0.preview.1

**Comment by rimwerda1996** [comment]
hello, i'm trying to use redmine api in my application, can you confirm if this code works correctly?

**Comment by stolp** [comment]
Yes, for my purposes this library yields good results - your mileage may vary.

Please don't hijack issue threads.

**Comment by eticco** [comment]
I had the same problem yesterday. You can find the explanation and a possible workaround in #306.

Another approach could be to extend Redmine, but I know nothing about RubyOnRails.

**Comment by stolp** [comment]
@ettico
Thanks a lot for the pointer! I am going to fork and check this out.
